src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs

```diff
@@ -262,6 +262,12 @@ public enum HWType
         NPUSR = 0x13000,
     }
 
+    public enum ServiceEventType
```

Comment:
It is public enum, so need comment for event type.
```
    /// <summary>
    /// Event types of machine learning service.
    /// </summary>
    /// <since_tizen> ?? </since_tizen>
    public enum ServiceEventType
    {
        /// <summary>
        /// New data is processed from machine learning service.
        /// </summary>
        NewData = 1,
    }
```

Reply:
Let's consider moving this enum to Service.cs
ServiceEventType > EventType in Service class

Reply:
```
/TizenFX/src/Tizen.MachineLearning.Inference/Interop/Interop.Nnstreamer.cs(336,61): error CS0426: The type name 'EventType' does not exist in the type 'Interop.Service' [/TizenFX/src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference.csproj]
```

There is a build error when moves enum to Service.cs

I found below code not to make enum public and use it with interop definition

https://github.com/niley7464/TizenFX/blob/080634f97cd33da9ebdd28cf5626767ed8a39c99/src/Tizen.Messaging.Push/Interop/Interop.PushClient.cs#L26

https://github.com/niley7464/TizenFX/blob/080634f97cd33da9ebdd28cf5626767ed8a39c99/src/Tizen.Messaging.Push/Tizen.Messaging.Push/PushImpl.cs#L152

How about this one?

Reply:
Updated with your suggestion to use the whole namespace
Thank you :)